lib/rubocop/cop/layout/multiline_array_brace_layout.rb

```diff
@@ -92,12 +92,12 @@ class MultilineArrayBraceLayout < Cop
         include MultilineLiteralBraceLayout
 
         SAME_LINE_MESSAGE = 'Closing array brace must be on the same line as ' \
-          'the last array element when opening brace is on the same line as ' \
-          'the first array element.'.freeze
+          'the last array element when the opening brace is on the same line ' \
```

Comment:
I guess you should add one `the` before `Closing` as well.

Reply:
@bbatsov Good point, just added that to each one.